homeassistant/const.py

```diff
@@ -1173,8 +1173,28 @@ class UnitOfVolumeFlowRate(StrEnum):
 )
 """Deprecated: please use UnitOfVolumeFlowRate.CUBIC_FEET_PER_MINUTE"""
 
-# Area units
-AREA_SQUARE_METERS: Final = "m²"
+
+class UnitOfArea(StrEnum):
+    """Area units."""
+
+    SQUARE_METERS = "m²"
```

Comment:
Please ensure that you update existing core integrations to use the new enum.

The deprecated string constant `AREA_SQUARE_METERS` should have a test in https://github.com/home-assistant/core/blob/dev/tests/test_const.py

Reply:
Indeed it does, OK cool that was the thing I was missing / didn't understand if I needed to update them.

Reply:
Updated